### DISCUSSION TITLE
add optino to print in the right click context menu

### DISCUSSION THREAD

**Discussion by dzwdev** [linuxmint/discussions#130] [Ideas]
in Windows, when selecting multiple .doc files and right click on them there is an option to print all those files in that menu instead of opening each single file separately. i hope that you add this option in Linux mint nemo explorer.

**Comment by rcalixte** [comment]
This is now available as an Action [here](https://cinnamon-spices.linuxmint.com/actions/view/4). You can download and install it manually or wait until the next release of Linux Mint (21.3, hopefully some time in December) where there will be built-in support for installation and management of Actions.

Just a note that installing it manually also means that it would not be automatically updated if there are updates and it would not have localization support if you wanted translations. If you want either of those things, hopefully you can wait a few more weeks.
